keylime/cloud_verifier_common.py

```diff
@@ -341,7 +341,9 @@ def process_get_status(agent):
 
 
 def notify_error(agent, msgtype='revocation'):
-    if not config.getboolean('cloud_verifier', 'revocation_notifier'):
+    send_mq = config.getboolean('cloud_verifier', 'revocation_notifier')
+    send_webhook = config.getboolean('cloud_verifier', 'revocation_notifier_webhook')
```

Comment:
I think we need a 3rd parameter here, which is the default, in case someone has an old keylime.conf after package upgrade....

Reply:
The default should be 'False'.

Reply:
@THS-on please address @stefanberger comment, so I can test and merge

Reply:
done